oonib.conf.example

```diff
@@ -28,7 +27,17 @@ main:
     debug: false
     stale_time: 3600
 
-    tor_hidden_service: true
+    tor_hidden_service: false
+    tor_datadir: test_datadir
+
+    bouncer_endpoints:
+    - {type: tls, port: 10443, cert: "private/ssl-key-and-cert.pem"}
+    - {type: tcp, port: 10080}
+    - {type: onion, hsdir: bouncer}
+
+    collector_endpoints:
+    - {type: tls, port: 11443, cert: "private/ssl-key-and-cert.pem"}
+
```

Comment:
It would be ideal if we could keep backward compatibility with old configuration files.

I am thinking that instead of having the endpoints be a list we could expand them to something like:
```
bouncer_tls_port: XXX
bouncer_tls_cert: XXX
bouncer_hsdir: XXX

collector_tls_port: XXX
collector_tls_cert: XXX
etc.
```